lib/nerves_motd.ex

```diff
@@ -131,9 +131,12 @@ defmodule NervesMOTD do
     [_total, _used, _free, _shared, _buff, _available] = runtime_mod().memory_usage()
   end
 
-  @spec load_average() :: String.t()
+  @spec load_average() :: iolist()
   defp load_average() do
-    runtime_mod().load_average()
+    case runtime_mod().load_average() do
+      [a, b, c] -> :io_lib.format("~s ~s ~s", [a, b, c])
```

Comment:
```suggestion
      [a, b, c | _] -> [a, " ", b, " ", c]
```
How about returning all 5 items and then this just prints the first 3. I don't think `:io_lib.format` is needed when making the `iolist` is just concatenation.